### PR TITLE
Remove Get Involved frontend rendering code from Whitehall

### PR DESCRIPTION
**Pull request by Tetrino**
Removes rendering code and associated helper functions from Whitehall
now that the rendering of the Get Involved page is handled by Government
Frontend.

Routes are left as is due to some history of unexpected behaviour when
removing Whitehall routing.

https://trello.com/c/aSeKkP6v/2718-remove-government-get-involved-rendering-code-from-whitehall-frontend-5

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Comment by ChrisBAshton** [comment]
This has now been tested on Integration 👍

File: features/step_definitions/take_part_pages_steps.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Given(/^there are some take part pages for the get involved section$/) do
```
Comment (benjamineskola) [inline features/step_definitions/take_part_pages_steps.rb:1]: Not a blocking review, but the naming here is a bit confusing: `page1`/`page2`/`page3` but ordering `2`, `3`, `1`. Could the names be made more clear regarding how they're arranged, perhaps?

File: features/step_definitions/take_part_pages_steps.rb [hunk 2]
```diff
@@ -34,35 +34,36 @@
...
-    expect(take_part_page.title).to eq(take_part_headings[idx])
+    expect(take_part_page.title).to eq(take_part_pages[idx])
   end
 end
 
 When(/^I remove one of the take part pages because it's not something we want to promote$/) do
   visit admin_get_involved_path
   click_on "Take part pages"
 
-  click_on @the_take_part_pages_in_order[0].title
```
Comment (benjamineskola) [inline features/step_definitions/take_part_pages_steps.rb:50]: If `@the_take_part_pages_in_order` is no longer being referenced here, can the other references/definitions be removed too, or do other tests depend on it still?

**Review by benjamineskola — APPROVED** [review]

File: features/step_definitions/take_part_pages_steps.rb (hunk 2 above)
Comment (ChrisBAshton) [reply features/step_definitions/take_part_pages_steps.rb:50]: It's still used in the `Then(/^I see the take part pages in my specified order including the new page on the admin view$/) do` block.

File: features/step_definitions/take_part_pages_steps.rb (hunk 1 above)
Comment (ChrisBAshton) [reply features/step_definitions/take_part_pages_steps.rb:1]: I completely agree - was going to attempt a refactor, but didn't get around to it. Not wanting to stall this any longer (I'm on leave until Wednesday), going to merge this for now, maybe this entire code will be deleted before a refactor is due.

**Pull request merged**